notebooks/tfx_pipelines/walkthrough/labs/tfx_walkthrough_vertex.ipynb

```diff
@@ -990,16 +990,19 @@
    "metadata": {},
```

Comment:
<p>Was this moved here mistakenly?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/329/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='101'/>